### ISSUE TITLE
SunSDR2DX Crashes When Connected to Amplifier

### ISSUE THREAD

**Issue by W2PH** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#104]
Running SunSDR2DX and EESDT Alpha 10.  On voice peaks when used with an amplifier will freeze and require a power button cycle to return to normal.  All I/O to the radio is treated with split ferrites including ethernet.  This occurs even on a dummy load with 1:1 SWR.  None of this happens in EESDR2.  I am using the PC audio for transmit and receive.  It happens on all bands and seems to be related to the RF output level. (It seems to freeze on voice peaks).
Ed W2PH

**Comment by williamscody** [comment]
Ed - does this issue have any relationship to the "zoom level" of the waveform?  I.e., does it only occur when zoomed in?

**Comment by W2PH** [comment]
I will check tomorrow and let you know.  I never really change screen zoom level, but I will check it in the morning.  I have the eesdr3 screen reduced to fit on half a 50” monitor.  

Ed

**Comment by N8SDR007** [comment]
This sounds more like a RFI issues getting into the computer and causing the scenario.

 and yes, you may not have had it before, but one thing I noticed and have opened a report on is an increase in IMD with this V3 0.10.0 release, which means more harmonics present to cause more RFI issues then earlier releases.

Quick test, drop drive level out to amp so that you're not reaching the current power level by say 20-30% and see if it continues to freeze up. I have noted an increase in voice peaks as opposed to previous releases and I also think that has caused a role in the increased IMD

**Comment by W2PH** [comment]
Hi Rick,

I dropped transceiver power so Amp output went from 1500w out to abput 1100w and it appears that maybe it doesn’t cause the hangup.  It was a brief test but it does suggest that perhaps you are on to something.  I am back on eesdr2 since I am net control for a variety of nets and cannot afford another lockup.  I was hopeful someone else might be having the problem so it would get some attention from Roman and the guys.  Not having VAC is also a major problem for me here.  It just seems to be taking ages for periodic updates… ofc compared to my old flex, the updates from Expert are coming fast and furiously 😊

 

Thanks for your help.  Any idea when Alpha 11 is coming?

Cheers,

Ed W2PH

**Comment by W2PH** [comment]
Not sure what you mean by “zoomed in”.  I am displaying the software on half my screen so that could be considered zoomed in.  I do not see any correlation with screen size/zoom with the crashing problem in Tx.

Ed

**Comment by williamscody** [comment]
Ed - I mean horizontal zoom, which is adjusted by right-clicking and dragging on the frequency bar.  Zoom in by dragging to the right.

**Comment by W2PH** [comment]
Just checked it at both horizontal zoom extremes and I can see no difference

**Comment by IU4JNR** [comment]
Hi, i reported almost the same issue. Differences are that it doesn't happen specifically on voice's peaks but even on cw using 40ish watts out into a Acom1010 which then delivery about 400w. However, reading Rick observations about IMD, I strongly reduced the microphone input. I also tried inserting chockes and ferrites at every single cable entering the sun with no success.
Max

**Comment by W2PH** [comment]
Hi,

With regard to Rick’s comment, I did the same…reduced voice peaks and have ferrites on everything going to the SDR2DX.  I also rerouted the ethernet cable completely away from any RF energy sources and added ferrites to the PC and the Network router cables… all to no avail.  I do many traffic nets and cannot afford it to crash when I am net control.  For now I have gone back to EESDR2.  I do realize that EESDR3 is not even Beta yet, but aside from that issue everything else was working and I had hoped to continue using it.

Cheers,

Ed W2PH
